librz/core/cmd_eval.c

```diff
@@ -741,7 +583,43 @@ RZ_IPI RzCmdStatus rz_eval_readonly_handler(RzCore *core, int argc, const char *
 
 RZ_IPI RzCmdStatus rz_eval_spaces_handler(RzCore *core, int argc, const char **argv) {
 	const char *arg = argc > 1 ? argv[1] : "";
-	rz_config_list(core->config, arg, 's');
+	RzConfigNode *node;
+	RzListIter *iter;
+	arg = rz_str_trim_head_ro(arg);
```

Comment:
there's no need for this. The argument is trimmed by the parser, so you just need to take it without further processing.

Reply:
@officialcjunior you forgot this one.